Articles/Defining and Using Observables.md

```diff
@@ -0,0 +1,341 @@
+
+For more, see [Populating The Value Field](#populating-the-value-field).
+
+#### Definition Nuance: Event Class Observables Are Not Supported
+Defining an event class as an observable is not supported. This would be essentially redundant with the `type_uid` field, which already uniquely identifies an OCSF event's event class. In other words, a query for an event class's observable ID might as well query for event classes `type_uid`. Further, the OCSF community is trying to move away from observables by object, which this would be similar to, since these do no populate the `observable` object's `value` field.
+
+The only thing this sort of definition would enable would be to detect events that are in event class inheritance subtree at a finer grain than categories; a fairly esoteric use-case.
+
+## Using Observables
+Observables can be generated automatically, though of course can also be created manually. Creating observables automatically requires walking an event's structure along with a compiled schema, looking for observable definitions at each level of the structure, as well at each leaf (each primitive value).
+
+A concrete example of this code can be found in the [`Schema` class](https://github.com/ocsf/ocsf-java-tools/blob/main/ocsf-schema/src/main/java/io/ocsf/schema/Schema.java) in the [`ocsf/ocsf-java-tools` repo](https://github.com/ocsf/ocsf-java-tools). Look at the `enrich(Map<String, Object>, boolean, boolean)` method and follow it along. As an aside, this same class and approach can be used to add enum sibling values, and indeed can (and should) be done in the same pass as adding observables.
+
+Whether creating observables manually -- a part of mapping process -- or automatically, care must be taken while populating the `name` and `value` fields.
+
+### Populating Observable Path References
+The `observable` object's `name` attribute is an attribute path reference. See [Appendix 1: Attribute Paths](#appendix-1-attribute-paths) for details.
+
+### Populating the Value Field
+The `observable` object's `value` field should be populated for all primitive types (strings, numbers, booleans), _and_ for arrays of primitive types (see next paragraph). The `value` field is specifically not meant to be used for observable objects, nor for for arrays of objects.
+
+For arrays of primitive types, one `observable` object should be created for each element of the array with the `value` field being set to the array element's value.
+
+#### All Observable Values Are Strings
+The `observable` object's `value` is defined as a type `string_t`. (Specifically notice that the type of `value` is not the problematic `json_t` type, which means any type.) 
```

Comment:
I would remove the editorializing about `json_t` - remove 'problematic' as it will bring up questions as to why you feel that way, which you aren't addressing.